1.10/tutorials/dcos-101/app2.md

```diff
@@ -11,52 +11,41 @@ menu_order: 5
-   * Connect to the webapp (from your local machine) via `<Public-IP>:10000`. You should see a rendered version of the web page including the physical node and port app2 is running on.
-     * Use the web form to add a new Key:Value pair
-     * You can verify the new key was added in two ways:
-       1. Check the total number of keys using app1: `dcos task log app1`
-       2. Check redis directly
-          * SSH into node where redis is running: `dcos node ssh --master-proxy --mesos-id=$(dcos task  redis --json |  jq -r '.[] | .slave_id')`
-          * NOTE: This requires you to have the ssh-key required to connect to the machines added to your local SSH agent (e.g., via ssh-add my_public_key). Check the [documentation](/docs/1.10/administering-clusters/sshcluster/) for further details.
-       * Because redis is running in a Docker container, you need to list all Docker containers using the `docker ps` command to get the *ContainerID*.
-         * Connect to a bash session to the running container: `sudo docker exec -i -t CONTAINER_ID  /bin/bash`
-         * Start the redis CLI: `redis-cli`
-         * Check value is there: `get <newkey>`
+      
+      This should return you the raw HTML response from app2's webserver.
+  
+
+Accessing the app from within the cluster and viewing the raw HTML response proves our application is running, but in the real world you want to expose the app to the public. In the next part of this tutorial you will do exactly that. 
 
 # Outcome
- You have deployed a second app that uses the native Mesos containerizer. You used Marathon-LB to expose the app to the public and added a new key to redis using the web frontend.
+ You have deployed a second app that uses the native Mesos containerizer.
 
 # Deep Dive
-You have now deployed apps in two different ways: using Docker (app1) and natively (app2).
-Let us explore the differences in some more detail.
-DC/OS uses [containerizers](/docs/1.10/deploying-services/containerizers/) to run tasks in containers. Running tasks in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically. DC/OS supports the Mesos containerizer types DC/OS Universal container runtime and Docker containerizer.
+You have now deployed apps in two different ways: 
+
+1. Using Docker (app1)
+1. Using the native Universal Container Runtime (app2)
+
+Let's explore the differences in some more detail.
+
+DC/OS uses [containerizers](/docs/1.10/deploying-services/containerizers/) to run tasks in containers. Running tasks in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically. DC/OS supports three types of containerizers - the Mesos containerizer, the DC/OS Universal Container Runtime, and the Docker containerizer.
```

Comment:
As of 1.10, I believe we're only mentioning the Docker Containerizer and UCR. The older version of the Mesos containerizer isn't available in the UI.